src/test/kotlin/com/igorwojda/integer/pyramidgenerator/challenge.kt

```diff
@@ -4,7 +4,8 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 fun generatePyramid(n: Int): List<String> {
-    TODO("not implemented")
+    var currentLevel = if (n==2) n / 2 else (n - 1)
```

Comment:
The challenge instructions specify that this function should log to the console the generated pyramid.